### DISCUSSION TITLE
同步oracle的number类型的字段数据格式问题

### DISCUSSION THREAD

**Discussion by 17554262006** [apache/flink-cdc#856] [Q&A]
各位大佬,有个问题,我在同步oracle数据库时,字段类型为number类型的数据,同步的结果变成 {"scale":0,"value":"A+g="} 这种格式,请问有什么解决办法嘛?

**Comment by ashulin** [comment]
SQL API这个已经解决了。不过需要等待2.2版本发布，如果急得话可以先编译master版本。如果是同步为json格式，请参考`com.ververica.cdc.debezium.table.RowDataDebeziumDeserializeSchema#createDecimalConverter`的处理方式。

**Comment by lizhe6132** [comment]
properties.setProperty("decimal.handling.mode","STRING");
